src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/__tests__/fixtures.ts

```diff
@@ -0,0 +1,205 @@
+import { BackupSecondFactorModalContentQueryRawResponse } from "v2/__generated__/BackupSecondFactorModalContentQuery.graphql"
+import { CreateAppSecondFactorMutationResponse } from "v2/__generated__/CreateAppSecondFactorMutation.graphql"
+import { CreateBackupSecondFactorsMutationResponse } from "v2/__generated__/CreateBackupSecondFactorsMutation.graphql"
+import { CreateSmsSecondFactorMutationResponse } from "v2/__generated__/CreateSmsSecondFactorMutation.graphql"
+import { DeliverSecondFactorMutationResponse } from "v2/__generated__/DeliverSecondFactorMutation.graphql"
+import { DisableSecondFactorMutationResponse } from "v2/__generated__/DisableSecondFactorMutation.graphql"
+import { EnableSecondFactorMutationResponse } from "v2/__generated__/EnableSecondFactorMutation.graphql"
+import { TwoFactorAuthenticationQueryRawResponse } from "v2/__generated__/TwoFactorAuthenticationQuery.graphql"
+import { UpdateAppSecondFactorMutationResponse } from "v2/__generated__/UpdateAppSecondFactorMutation.graphql"
+import { UpdateSmsSecondFactorMutationResponse } from "v2/__generated__/UpdateSmsSecondFactorMutation.graphql"
```

Comment:
Where are these files? Presumably they need to be migrated & updated as well.

Reply:
These are located in that same fixture file. They were only being utilized in this`TwoFactorAuthentication.jest.tx` file, and now replicated for the `SettingsEditSettingsTwoFactor.jest.tsx` files.

Reply:
Sorry, I actually mean the corresponding front end components that these specs are referring to.